testprof/base64_test.c

```diff
@@ -7,7 +7,7 @@ int base64_test(void)
    unsigned long x, l1, l2, slen1;
 
 #if defined(LTC_BASE64)
-   const char special_case[] = {
+   const unsigned char special_case[] = {
```

Comment:
please also fix declaring `special_case` in the case where only `LTC_BASE64_URL` is defined